Source/MaestroAverage.cpp

```diff
@@ -155,9 +155,6 @@ void Maestro::Average(const Vector<MultiFab>& phi, BaseState<Real>& phibar,
             ParallelFor(base_geom.nr_fine, [=] AMREX_GPU_DEVICE(int r) {
                 if (ncell(lev, r) > 0) {
                     phisum_arr(lev, r) /= ncell(lev, r);
-                } else {
```

Comment:
Do you recall why you deleted the "else" case?

Reply:
I didn't think it was necessary since we don't have this "else" case anywhere else.

Reply:
I think we got away with only having the else case here since it's the final step... if there were bins that had no cartesian cells map to them, we could just ignore them until the very end where we filled the average with a semi-sensible value (the value from the bin inward).

I think for both spherical cases where we compute phisum we need this check.  The Cartesian version doesn't since there will always be cells that map to the radial bins.